packages/devtools_app/lib/main.dart

```diff
@@ -3,6 +3,7 @@
 // found in the LICENSE file.
 
 import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
```

Comment:
how much value are we getting from riverpod here? Using both provider and riverpod seems a bit redundant.